MekHQ/src/mekhq/gui/utilities/JScrollPaneWithSpeed.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2019-2024 - The MegaMek Team. All Rights Reserved.
+ *
+ * This file is part of MekHQ.
+ *
+ * MekHQ is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MekHQ is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MekHQ. If not, see <http://www.gnu.org/licenses/>.
+ */
+package mekhq.gui.utilities;
+
+import java.awt.*;
+
+import javax.swing.*;
+
+import megamek.client.ui.swing.GUIPreferences;
+
+/**
+ * It's a JScrollPane that manages its scrollspeed based on the UI scale
+ */
+public class JScrollPaneWithSpeed extends JScrollPane {
+    final static int BASE_INCREMENT = 16;
+
+    /**
+     * @see JPanel#JPanel()
+     */
+    public JScrollPaneWithSpeed() {
```

Comment:
Would it make sense to use overrides in this class?

Reply:
You mean `@Override`?  I didn't know that worked on constructors

Reply:
Sorry, that was just poor placement on my part. I had to stick the comment somewhere, but it was more a general comment on the class as a whole.

Reply:
Conversation can be ignored.